### ISSUE TITLE
Accessibility: Conformance to WCAG (Web Content Accessibility Guidelines)

### ISSUE THREAD

**Issue by NomadicCore** [imfing/hextra#921]
**Feature Description**

Clear conformance to the [Web Content Accessibility Guidelines International Standard](https://www.w3.org/WAI/standards-guidelines/wcag/) as a way of making websites more accessible to people with disabilities.

**Problem/Solution**

I'm sure Hextra already has accessibility features but it would be good to compare to the WCAG standards and confirm whether it is fully or partially compliant. This could also lead to a road map for feature requirements to ensure full standards.

This is important as people with disabilities should be catered as first class citizens.

**Alternatives Considered**

People would individually need to assess site accessibility using the Hextra theme which would create a lot of duplication.

**Additional Context**

I am currently preparing a new website using the Hextra theme for AerynOS (a Linux distribution currently in alpha). We have several members of our team with accessibility issues so is a topic that is close to home.

Edit: As an aside, a glossary is something that is recommended as part of the WCAG and I know something you've recently delivered upon (https://www.w3.org/WAI/WCAG22/Techniques/general/G62).
